source/conf.py

```diff
@@ -19,9 +19,9 @@
 
 # -- Project information -----------------------------------------------------
 
-project = u'perun-doc'
-copyright = u'2020, Hyperledger'
-author = u'Hyperledger'
+project = u'Perun Framework'
+copyright = u'2020, Perun Framework Contributors'
+author = u'Perun Framework Contributors'
```

Comment:
@tkuhrt, @ryjones: Is there a recommendation from you on how copyright statements of Hyperledger (Labs) documentation should look like? The way this is all set up, we have common ownership as no copyright is transferred. So the proposed change fits quite well IMO, even though, strictly speaking, there can be and quite often is a difference between copyrights-holders and authors... 

@matthiasgeihs , IIRC, [_templates/footer.html](https://github.com/hyperledger-labs/perun-doc/blob/master/source/_templates/footer.html) needs also to be patched accordingly.

Reply:
We (Hyperledger) has been fairly liberal on this point. We're more concerned that the license (Apache 2, CC-4-BY) is correctr.

Reply:
As Ry mentioned, we have been fairly liberal; however, there is this [wiki page](https://wiki.hyperledger.org/display/TSC/Copyright+and+License+Policy) that talks about Copyright and License Policy.